lib/gem2rpm/package.rb

```diff
@@ -1,4 +1,6 @@
 require 'delegate'
+# require 'rubygems' to avoid NameError at Ruby 1.8.7.
+require 'rubygems' unless defined?(Gem)
```

Comment:
Yes, and that is the point. The test fails, but I don't think you can trigger this behavior in real life usage, so I'd prefer to remove these two lines ...